File: web/Maevent.API/Models/EventModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Maevent.API.Models
{
    public class EventModel
    {
        [Required]
        [MinLength(5)]
        [MaxLength(30)]
        public string Name { get; set; }
        [Required]
        public string Place { get; set; }
        public string AddressStreet { get; set; }
        public string AddressPostCode { get; set; }
        [Required]
        public DateTime BeginTime { get; set; }
        [Required]
        public DateTime EndTime { get; set; }
        public Boolean Rsvp { get; set; }

        [Required]
        public int hostId { get; set; }
    }
}

File: mobile/app/src/main/java/com/devmarcul/maevent/apis/models/EventModel.java
package com.devmarcul.maevent.apis.models;

import com.devmarcul.maevent.apis.MaeventApiModel;

public class EventModel extends MaeventApiModel {
    public String Name;
    public String Place;
    public String AddressStreet;
    public String AddressPostCode;
    public String BeginTime;
    public String EndTime;
    public boolean Rsvp;
    public int HostId;
}
